docs/Makefile

```diff
@@ -37,7 +37,41 @@ help:
 html-noplot:
 	$(SPHINXBUILD) -D plot_gallery=0 -b html $(SPHINXOPTS) "$(SOURCEDIR)" "$(BUILDDIR)/html"
 
-download:
+download-noplot:
```

Comment:
Could you add a comment that explains why we have download-noplot and download? For example, files that are hosted on the website should go into download-noplot whereas files which are only required to run a tutorial go into download.